specs/core/discovery.html

```diff
@@ -419,6 +419,39 @@ <h2>General Discovery Methods</h2>
         </p>
       </section>
 
+      <section id="wellKnown">
+        <h2>Well-known URI Bootstrapping</h2>
+
+        <p class="conformance">
+          An OSLC Server MAY serve a Root Services document at the
+          <code>/.well-known/oslc/rootservices.xml</code> URI [[!WELL-KNOWN]] if the Server provides access to such a
+          document. The Root Services document SHALL conform to the [[!ROOT-SERVICES]] specification. The server SHALL
+          serve it using the RDF/XML-ABBREV format with the MIME type of <code>application/xml</code> or
```

Comment:
I'm not keen on the way this is phrased - it seems as if it would make future format additions difficult. I think it should say something like:
"If  no specific content type is requested, or if the requested content type is  <code>application/xml</code> or <code>application/rdf+xml</code>, the server SHALL return the Root Services document using the RDF/XML-ABBREV format. Servers MAY return other formats in response to other requested content types."